core/controllers/story_editor.py

```diff
@@ -90,6 +90,16 @@ def put(self, story_id):
         self._require_valid_version(version, story.version)
 
         commit_message = self.payload.get('commit_message')
+
+        if commit_message is None:
```

Comment:
Why the `none` check for only the story editor?

Reply:
In other places, there is a strict enforcement in the retrieval of the model, or the model is checked to be None prior to retrieval. As such, this was the only place where commit message would actually be None in testing.